butte/src/table.rs

```diff
@@ -55,13 +55,17 @@ where
     pub fn get<'a, T: Follow<'a> + 'a>(
         &'a self,
         slot_byte_loc: VOffsetT,
+        default: Option<T::Inner>,
     ) -> Result<Option<T::Inner>, Error> {
         let v_offset = self.vtable()?.get(slot_byte_loc)?;
         if let Some(v_offset) = v_offset {
             let o = v_offset as usize;
+            if o == 0 {
```

Comment:
`o_O` not really a huge fan of the `o == 0` spelling, can we change `o` to `offset`? :)

Reply:
Sure